velox/vector/arrow/Bridge.cpp

```diff
@@ -1399,10 +1484,37 @@ VectorPtr createTimestampVector(
   const auto* rawNulls = nulls->as<const uint64_t>();
```

Comment:
nulls is initialized as `BufferPtr nulls = nullptr;` in `importFromArrowImpl()` and will not be updated is the null_count is zero. So I'am afraid it will cause crash to cast a nullPtr to rawNulls and access to it.

Reply:
Nice catch, thanks. Fixed in this PR, and I will extract this fix to a separate PR.

Reply:
@ViggoC Fixing this issue in https://github.com/facebookincubator/velox/pull/8955.